benchmarks/query_benchmarks/query_aggregate/benchmark.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import Count
+
+from ...utils import bench_setup
+from .models import Book
+
+
+class QueryAggr:
+    def setup(self):
+        bench_setup(migrate=True)
+
+    def time_aggregate(self):
```

Comment:
This benchmark is currently in the 800μs range.

Shall we repeat it say 100x to get it into the 80ms range? (Hopefully I've got my maths correct!)